concrete/single_pages/dashboard/system/permissions/maintenance.php

```diff
@@ -7,7 +7,8 @@
     <?php echo $this->controller->token->output('update_maintenance')?>
     <fieldset>
         <div class="form-group">
-            <div class="radio"><label><?=$form->radio('site_maintenance_mode', '1', $site_maintenance_mode)?> <span><?=t('Enabled')?></span></label></div>
+            <label>Only Dashboard Works</label>
```

Comment:
Can you wrap this string in a `t('...')` function call so that translation can work with this string?

Reply:
Ooops, will fix this asap! :)